src/test/java/io/redis/examples/SortedSetsExample.java

```diff
@@ -0,0 +1,111 @@
+//ExAMPLE: ss_tutorial
```

Comment:
Turns out, case matters. Please commit this change and then it's good to go.

```suggestion
//EXAMPLE: ss_tutorial
```